### ISSUE TITLE
Fullscreen button goes away on large screens when full screen is activated

### ISSUE THREAD

**Issue by BjoernRave** [xiaolin/react-image-gallery#439]
Hey, first up I really like your library. 

Just one thing which I would like is to be able to exit the fullscreen by clicking the fullscreen button a second time

**Comment by xiaolin** [comment]
The fullscreen button is still available, you can click again to close it.

**Comment by BjoernRave** [comment]
really doesnt work for me, but I will check again

**Comment by djorg83** [comment]
For me the image is larger than the screen, and its forcing the fullscreen close button out of the viewable area.  The page is not scrollable, so I cannot get to the button.

**Comment by djorg83** [comment]
I solved it with the following css rule.

``` css
.image-gallery-slide img {
    width: 100vw;
    max-height: 100vh;
}
```

**Comment by xiaolin** [comment]
That will stretch your image and it won't maintain aspect ratio. I suggest keeping width auto and prioritize based on height during fullscreen.

something like

```scss
.image-gallery-content.fullscreen  {
  .image-gallery-slide img {
    width: auto;
    max-height: 100vh;
  }

  .image-gallery-image {
    text-align: center;
  }
}
```

I'll mark this as a bug as full-screen shouldn't extend past the height of the entire screen.

**Comment by BjoernRave** [comment]
I can see the fullscreen button, but it doesnt do anything. I am using custom icons: https://plezzles.com/places/ck09xetsz01li07462m0jbwnh

**Comment by xiaolin** [comment]
@BjoernRave can you show how you're overriding the `renderFullscreenButton` prop?

**Comment by BjoernRave** [comment]
sure:

```
renderFullscreenButton={(onClick, disabled) => (
          <FullScreenButton onClick={onClick} disabled={disabled}></FullScreenButton>
        )}
```

`FullScreenButton` is a styled-component

**Comment by xiaolin** [comment]
@BjoernRave You're disabling the button once it is in fullscreen mode. I would use the second argument for toggling different icons if you need. E.g.

```jsx
renderFullscreenButton={(onClick, isFullscreen) => (
     <FullScreenButton isFullscreen={isFullscreen} onClick={onClick}></FullScreenButton>
)}
```

```jsx
const FullScreenButton = styled.button`
  position: absolute;
  bottom: 10px;
  right: 0;
  z-index: 1;
  background: url('../../static/icons/fullScreen.svg');
  height: 50px;
  width: 50px;
  border: none;
  color: white;
  font-size: 35px;
  cursor: pointer;
  transition: all linear 0.1s;

  ${props => props.isFullscreen && `background: url('.../../static/icons/closeFullScreen.svg);`}

  :hover {
    transform: scale(0.9);
  }

  :active {
    transform: scale(0.85);
  }
`
```

**Comment by xiaolin** [comment]
fixed and released in version >= v1.0.0
